tests/baselines/reference/regexpExecAndMatchTypeUsages(strict=true).errors.txt

```diff
@@ -45,6 +53,9 @@ tests/cases/compiler/regexpExecAndMatchTypeUsages.ts(14,5): error TS2412: Type '
         }
         else {
             execResult = matchResult
+            ~~~~~~~~~~
+!!! error TS2375: Type 'RegExpMatchArray' is not assignable to type 'RegExpExecArray' with 'exactOptionalPropertyTypes: true'. Consider adding 'undefined' to the types of the target's properties.

```

Comment:
This sure is a weird error message since `undefined` doesn't show up at all in the source type. I would guess that this is a bug in `exactOptionalPropertyTypes`.